core/src/test/java/io/questdb/test/cairo/mv/CreateMatViewTest.java

```diff
@@ -659,6 +659,28 @@ public void testCreateMatViewSampleByWithOffset() throws Exception {
         });
     }
 
+    @Test
+    public void testCreateMatViewTsAlias() throws Exception {
```

Comment:
The following test fails
```java
    @Test
    public void testCreateMatViewGroupByWithTsAlias() throws Exception {
        assertMemoryLeak(() -> {
            createTable(TABLE1);
            final String query = "select ts ts1, v_max v_max1 from (select timestamp_floor('30s', ts) as ts, max(v) as v_max from " + TABLE1 + ")";
            execute("create materialized view test as (" + query + ") partition by week");
            assertMatViewDefinition("test", query, TABLE1, 30, 's');
            assertMatViewMetadata("test", query, TABLE1, 30, 's');
        });
    }
```

It looks like we may need to consider aliases assigned by "plain select" models.